src/mobile/apps/auction_support/stylesheets/index.styl

```diff
@@ -5,6 +5,52 @@
 @import '../components/stylus_lib'
 @import '../components/bid_numbers'
 
+// checkbox-size = 20px
```

Comment:
can these commented-out styles be removed?

Reply:
I committed them by accident and meant to remove them but did the wrong `git reset` command. I'll do it but the mobile pr also fixes this anyways.